modules/OCPP201/conversions.cpp

```diff
@@ -827,5 +827,27 @@ to_everest_certificate_status(const ocpp::v201::AuthorizeCertificateStatusEnum s
     }
 }
 
+types::ocpp::OcppTransactionEvent
+to_everest_ocpp_transaction_event(const ocpp::v201::TransactionEventRequest& transaction_event) {
+    types::ocpp::OcppTransactionEvent ocpp_transaction_event;
+    switch (transaction_event.eventType) {
+    case ocpp::v201::TransactionEventEnum::Started:
+        ocpp_transaction_event.transaction_event = types::ocpp::TransactionEvent::Started;
+        break;
+    case ocpp::v201::TransactionEventEnum::Updated:
+        ocpp_transaction_event.transaction_event = types::ocpp::TransactionEvent::Updated;
+        break;
+    case ocpp::v201::TransactionEventEnum::Ended:
+        ocpp_transaction_event.transaction_event = types::ocpp::TransactionEvent::Ended;
+        break;
+    }
+    ocpp_transaction_event.evse_id = transaction_event.evse.value().id;
+    ocpp_transaction_event.connector = 1;
```

Comment:
if the connectorId is available this should probably use the one from transaction_event.evse.value().connectorId

Reply:
not sure where this change comes from. I don't remember changing this fine.
a merge issue perhaps?